vitrina/comments/views.py

```diff
@@ -47,11 +47,15 @@ def post(self, request, content_type_id, object_id):
                 comment.rel_object_id = new_request.pk
 
             elif form.cleaned_data.get('status'):
-                comment.type = Comment.STATUS
-                obj.status = form.cleaned_data.get('status')
-                obj.comment = comment.body
-                obj.save()
-                set_comment(Request.STATUS_CHANGED)
+                if obj.status == form.cleaned_data.get('status'):
+                    messages.error(request, 'Dabartinė būsena sutampa su jūsų pateikta')
```

Comment:
This should happen in `form_is_valid`, this is a validation thing and nothing should be saved if there is an error.